seqr/utils/vcf_utils.py

```diff
@@ -0,0 +1,80 @@
+import re
+
+from collections import defaultdict
+
+from seqr.utils.middleware import ErrorsWarningsException
+from seqr.utils.file_utils import file_iter
+
+BLOCK_SIZE = 65536
+
+EXPECTED_META_FIELDS ={
+    'INFO': {
+        'AC': 'Integer',
+        'AN': 'Integer',
+        'AF': 'Float'
+    },
+    'FORMAT': {
+        'AD': 'Integer',
+        'DP': 'Integer',
+        'GQ': 'Integer',
+        'GT': 'String'
+    }
+}
+
+REQUIRED_HEADERS = ['#CHROM', 'POS', 'ID', 'REF', 'ALT', 'QUAL', 'FILTER', 'INFO', 'FORMAT']
+
+
+def _validate_vcf_header(header):
+    errors = []
+    if len(header) <= len(REQUIRED_HEADERS):
```

Comment:
so as I said previously in my review of this PR, we allow extra headers to be present, and we don't want to detect and extra column as a sample. You need to maintain the current behavior for determining the samples. Please see my proposed solution from my previous comment: https://github.com/broadinstitute/seqr/pull/2945#discussion_r968555579

Reply:
I think the `FORMAT` is not a robust delimitation.

Reply:
Okay, I checked the VCF standard, and it says a `FORMAT` column follows any additional columns, then an arbitrary number of sample IDs.

Reply:
we are going to maintain current behavior, which considers all columns after the FORMAT column to be samples